src/client.jl

```diff
@@ -9,11 +9,11 @@ results. Should only be used directly for advanced workflows. See [`DaskExecutor
 instead for normal usage.
 
 # Fields
-- `nodes::Dict{String, DispatchNode}`: maps keys to their dispatcher `DispatchNode`
+- `nodes::Dict{String, DispatchNode}`: maps previously submitted byte keys
```

Comment:
I will also rename this from`nodes` to `keys` when I update this Dictionary to a Set as part of another MR. (This docstring is outdated: nodes is actually a `Dict{Array{UInt8, 1}, Void}` right now, will just update it to reflect the correct current type and will fix the Types and name in the other PR)